cleaner/environment_matrix/classes/clean.php

```diff
@@ -30,6 +30,9 @@
     die('Direct access to this script is forbidden.'); // It must be included from a Moodle page.
 }
 
+require_once('../../../../../lib/adminlib.php');
+require_once(__DIR__ . '/../../../../../config.php');
```

Comment:
this looks wrong? this would bootstrap moodle which should already be done long ago

Reply:
Remnants from trying some stuff. Ill remove